lib/allora/lib/allora-stack.ts

```diff
@@ -0,0 +1,104 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as s3deploy from 'aws-cdk-lib/aws-s3-deployment';
+import * as fs from 'fs';
+import * as path from 'path';
+// import * as sqs from 'aws-cdk-lib/aws-sqs';
+
+
+export interface AlloraStackProps extends cdk.StackProps {
+  amiId: string;
+  instanceType: string;
+  vpcMaxAzs: number;
+  vpcNatGateways: number
+  vpcSubnetCidrMask: number;
+  resourceNamePrefix: string;
+}
+
+
+export class AlloraStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: AlloraStackProps) {
+    super(scope, id, props);
+
+    // Parameters
+    const region = props?.env?.region || 'us-east-1';
```

Comment:
You should remove these default values like `us-east-1` since it is duplicating the default values provided by `allora.ts` when it is creating this stack.